src/components/Error/Error.jsx

```diff
@@ -36,7 +36,7 @@ const Error = ({ errors, showBackButton, canDismissAlert, router }) => {
 Error.propTypes = {
   canDismissAlert: PropTypes.bool,
   errors: PropTypes.shape({
-    errorText: PropTypes.string.isRequired,
+    errorText: PropTypes.array.isRequired,
```

Comment:
can you use `arrayOf` and define the array, like we do with objects? there are examples of it in the code if you need it.

Reply:
done